source/javascripts/components/YmlEditorHeader/YmlEditorHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import React  from "react";
+import { Button, Flex, Text } from "@bitrise/bitkit";
+import { useMediaQuery } from "@bitrise/bitkit/lib/esn/hooks";
+
+import WorkflowRecipesLink from "../workflow-recipes/WorkflowRecipesLink/WorkflowRecipesLink";
+
+type YmlEditorHeaderProps = {
+	url: string;
+	usesRepositoryYml?: boolean;
+}
+const YmlEditorHeader = ({ url, usesRepositoryYml}: YmlEditorHeaderProps): JSX.Element => {
+	const match = useMediaQuery(["848px"]);
```

Comment:
I'm not sure how we are handling media queries in workflow editor, but I'm sure it would be good to have an agreement about the breakpoint, move it to a variable and use that variable here, but we can do it later.
WDYT @komaromia ?

Otherwise it LGTM 👍🏻